include/seqan3/alignment/pairwise/detail/policy_affine_gap_with_trace_recursion.hpp

```diff
@@ -78,9 +78,11 @@ class policy_affine_gap_with_trace_recursion : protected policy_affine_gap_recur
         diagonal_score = (diagonal_score < vertical_score)
                            ? (best_trace = previous_cell.vertical_trace(), vertical_score)
                            : (best_trace |= previous_cell.vertical_trace(), diagonal_score);
-        diagonal_score = (diagonal_score < horizontal_score)
-                           ? (best_trace = previous_cell.horizontal_trace(), horizontal_score)
-                           : (best_trace |= previous_cell.horizontal_trace(), diagonal_score);
+        diagonal_score =
+            (diagonal_score < horizontal_score)
+                ? (best_trace = previous_cell.horizontal_trace() | (best_trace & trace_directions::carry_up_open),
```

Comment:
`(best_trace & trace_directions::carry_left_open)` is not needed in the above (L 79) case, because the horizontal trace is always set